CMakePresets.json

```diff
@@ -0,0 +1,23 @@
+{
+  "version": 3,
```

Comment:
That makes in usable only with CMake 3.21+.

And CMake 3.19 does fails _silently_  when `cmake --preset dev-mode`.

Reply:
Hm, I see. Do you have a suggestion? I don't think we can do much about it, except hope that noone tries to use this with 3.19 or 3.20.

Reply:
As it is a developer-specific feature, I'm OK with the current patch.